src/server/pps/server/api_server.go

```diff
@@ -2221,7 +2224,7 @@ func (a *apiServer) StartPipeline(ctx context.Context, request *pps.StartPipelin
 		return nil, err
 	}
 
-	if err := a.updatePipelineState(pachClient, request.Pipeline.Name, pps.PipelineState_PIPELINE_RUNNING); err != nil {
```

Comment:
After creating a helper to mark the pipeline as stopped, we only used this helper (`updatePipelineState`) here to mark a pipeline as running, so I simplified the helper and named it as such.

Reply:
These helpers should be the same though shouldn't they?